tests/test_etl.py

```diff
@@ -90,7 +88,8 @@ def test_etl(self):
                 staging_directory=td,
                 content_type_patterns=["application/json"],
                 filename_patterns=["*.json"],
-                dss_client=MockDSSClient()
+                dss_client=MockDSSClient(),
+                http_client=MockDSSClient()
```

Comment:
nit: Separate out the `MockDSSClient` into a parent `MockHTTPClient` class and `MockDSSClient` child class for clarity

Reply:
Done.